runtimes/bundles/org.jboss.tools.rsp.server.minishift/src/main/java/org/jboss/tools/rsp/server/minishift/servertype/impl/MinishiftServerTypes.java

```diff
@@ -22,22 +22,31 @@
 	public static final String CDK_3X_ID = "org.jboss.tools.openshift.cdk.server.type.minishift.cdk.3x";
 	public static final String CDK_3X_NAME = "CDK 3.x";
 	public static final String CDK_3X_DESC = "A server adapter capable of controlling a CDK 3.x runtime instance.";
+	
+	public static final String CRC_1X_ID = "org.jboss.tools.openshift.cdk.server.type.minishift.crc.1x";
+	public static final String CRC_1X_NAME = "CRC 1.x";
+	public static final String CRC_1X_DESC = "A server adapter capable of controlling a CRC 1.x runtime instance.";
 
 	public static final String RUNTIME_MINISHIFT_17_ID = "org.jboss.tools.openshift.cdk.server.runtime.type.minishift.17";
 	public static final String RUNTIME_CDK_30_ID = "org.jboss.tools.openshift.cdk.server.runtime.type.cdk.30";
 	public static final String RUNTIME_CDK_32_ID = "org.jboss.tools.openshift.cdk.server.runtime.type.cdk.32";
+	public static final String RUNTIME_CRC_10_ID = "org.jboss.tools.openshift.cdk.server.runtime.type.crc.10";
```

Comment:
JBT has already been published with runtime type id = org.jboss.tools.openshift.crc.server.runtime.type.crc.100  and so the runtime type in stacks.yaml has to match that. Working on it in jbt land right. RSP will adjust accordingly.